modules/server/src/network/types.ts

```diff
@@ -0,0 +1,13 @@
+import { GraphQLSchema } from 'graphql';
+
+// environment config
+export type NetworkAggregationConfigInput = {
+	graphqlUrl: string;
+	documentType: string;
+	displayName: string;
+};
```

Comment:
The comment here doesn't tell me anything new. You can use TSDoc on types as well to leave a description of the type in intellisense for future devs that use it.